src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -385,7 +385,6 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
                                   events: Seq[ExecutionEventEntry]) = {
     val currentSender = sender()
     val keyClone = callKey.retryClone
-    executionStore += keyClone -> ExecutionStatus.NotStarted
```

Comment:
I think this might have been causing the transient Retryable Spec failures. The cloned key was inserted in the ExecutionStore before the async DB persistence completed, which might lead to the call being started by the WA before it's actually in the DB.